Xamarin.Forms.Platform.Android/FastRenderers/LabelRenderer.cs

```diff
@@ -263,6 +263,24 @@ void UpdateFont()
 			}
 		}
 
+		void UpdateTextDecorations()
+		{
+			if (!Element.IsSet(Label.TextDecorationsProperty))
+				return;
+
+			var textDecorations = Element.TextDecorations;
+
+			if ((textDecorations & TextDecorations.Strikethrough) == 0)
```

Comment:
how do you feel about using `HasFlag()` instead? (here and there)

Reply:
It seems like the standard for this project is to avoid using `HasFlag` for performance reasons (although I do see it used in some places) and use bitwise ops instead, so that's why I went that route.

Reply:
> avoid using HasFlag for performance reasons

Reply:
you're right, we try to avoid HasFlags

also, we prefer handling the `true` case first, like this

```
if ((textDecorations & TextDecorations.Strikethrough) == TextDecoration.StrikeThrough)
```